### DISCUSSION TITLE
Trailing Slash lost query parameters on redirect

### DISCUSSION THREAD

**Discussion by renzit** [vercel/next.js#43248] [Help]
Hi, I have one doubt regarding a feature that is not working as expected.
When enabling trailing slash, it lost query parameters on redirect

yarn build && yarn export.
Generated export in live server:

access: /about-us?name=person
redirect: /about-us/

Expected redirect: /about-us/?name=person

I have next.config:
```
  experimental: {
    images: {
      unoptimized: true,
    },
  },
  output: 'standalone',
  staticPageGenerationTimeout: 1000,
  trailingSlash: false,
```
is something missing or misconfigured? 
is this a bug? 

When I use yarn dev instead of the export, it works as expected.
I would like to solve this in next, any ideas?

I also found some issues in this repo, but with no response. eg:https://github.com/vercel/next.js/issues/24483

**Comment by guitheengineer** [comment]
Should be working now on the latest version.
